apps/studio/src/server/modules/folder/folder.router.ts

```diff
@@ -20,6 +23,15 @@ export const folderRouter = router({
           parentId: parentFolderId ? String(parentFolderId) : null,
         })
         .executeTakeFirstOrThrow()
+        .catch((err) => {
+          if (get(err, "code") === "23505") {
+            throw new TRPCError({
+              code: "CONFLICT",
+              message: "A resource with the same permalink already exists",
+            })
+          }
+          throw err
+        })
```

Comment:
we need to duplicate this for `collection.router` otherwise, we get an ugly error message

Reply:
fixed